### ISSUE TITLE
Allow imports to be specified as org, api and version

### ISSUE THREAD

**Issue by gheine** [apicollective/apibuilder#699]
Currently imports require a uri pointing to the imported service.json, which binds the import to a particular instance of apibuilder (eg. app.apibuilder.io). Would be nice to be able to specify imports as a tuple of organization, api and version and resolve the import from the "local" apibuilder instance.

**Comment by mbryzek** [comment]
makes sense - and internally we just parse the URL anyway to pull out the org/app/version. I think the change is probably small - mainly to change import url validation to accept any host
